server/routes/api/delete/deleteUserHandler.ts

```diff
@@ -8,6 +9,32 @@ export default class Handler {
     this.dbHandler = new DBHandler();
   }
 
+  parentValidation = async(parentId: number): Promise<Validation> => {
+    const parentProfile = await this.dbHandler.getParent(parentId);
+    if (parentProfile.length === 0) {
+      return { isValid: false, message: 'Incorrect id' }
+    }
+
+    if (parentProfile[0].endTimestamp !== String(END_TIMESTAMP)) {
+      return { isValid: false, message: 'This user is not active' }
+    }
+
+    return { isValid: true }
+  }
+
+  babysitterValidation = async(babysitterId: number): Promise<Validation> => {
+     const babysitterProfile = await this.dbHandler.getBabysitter(babysitterId);
+    if (babysitterProfile.length === 0) {
```

Comment:
if you want to add if the array is empty then you can write
if (!babysitterProfile.length)